src/utils/isAppLaunched.ts

```diff
@@ -0,0 +1,3 @@
+export class AppState {
```

Comment:
Are you creating this class because you need it to be able to freeze the object ? Otherwise I think it is overkill to create a new class.

Reply:
Yes I need to freeze the object to make sure that the class won't be changed after the app was launched, do you think It's needed ? Because indeed, I could just set a variable 'isAppLaunched' to true and that would be it.

Reply:
No I think it is good practice that you lock this variable if you are able to.